ui/component/or-asset-viewer/src/or-edit-asset-panel.ts

```diff
@@ -204,7 +204,18 @@ export class OrEditAssetPanel extends LitElement {
 
     shouldUpdate(changedProperties: PropertyValues): boolean {
         if (changedProperties.has("asset")) {
-            this.changedAttributes = [];
+            const oldAsset = changedProperties.get("asset") as Asset;
+            if (oldAsset?.attributes) {
+                // Handles external attribute changes post saving phase.
+                this.changedAttributes = Object.values(oldAsset.attributes).filter(({ name, timestamp: oldTimestamp, value: oldValue }) => {
+                  if (this.asset.attributes && name) {
+                    return oldTimestamp !== this.asset.attributes[name].timestamp
+                  }
+                  return false;
+                }).map(({ name }) => name!);
+            } else {
+                this.changedAttributes = [];
+            }
```

Comment:
Might be better to work from the new asset and compare with old as an attribute could have been added or removed and this would not detect it, something like:

```suggestion
            const oldAsset = changedProperties.get("asset") as Asset;
            if (this.asset?attributes && oldAsset?.attributes) {
                // Handles external attribute changes post saving phase.
                this.changedAttributes = Object.values(this.asset.attributes).filter(({ name, timestamp }) => {
                  return oldAsset.attributes[name]?.timestamp !== timestamp
                }).map(({ name }) => name!);
            } else {
                this.changedAttributes = [];
            }
```